spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -18,14 +18,61 @@
       get :index, format: :json, featured: true
 
       expect(response.status).to eq 200
-      expect(json_last_skill["name"]).to eq("rspec")
+      expect(json_response_last_skill_name).to eq("rspec")
+    end
+  end
+
+  describe "create" do
+
+    it "creates a skill" do
+      post :create, format: :json, skill: { name: "created" }
+
+      expect(response.status).to eq 201
+      expect(json_response_created_skill_name).to eq("created")
+    end
+
+    it "creates a skill with full parameters" do
+      group = create(:group)
+      post :create, format: :json, skill: { 
+                                     name: "created", 
+                                     featured: true,
+                                     group: group.name
+                                   }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.